app/views/students/_former_students_table.html.erb

```diff
@@ -1,30 +1,32 @@
 <script type="text/javascript" src="/javascript/table.js"></script>
 <div id="man" class="col s12 m10 offset-m1">
-    <div class="card material-table">
+  <div class="card material-table">
     <div class="table-header">
-        <span class="table-title">Exbecarios</span>
-        <div class="actions">
-            <div class="col"><a class="btn-small disabled table-button" table-button-tag="VER">Ver</a></div>
-            <div class="col"><a class="btn-small disabled table-button" table-button-tag="CV">CV</a></div>
-            <div class="col"><a class="btn-small disabled table-button" table-button-tag="ELIMINAR"">Eliminar</a></div>
-            <div class="col"><%= link_to 'Crear', new_with_former_student_path, :class=>"btn-small"%></div>
-            <div class="col"><%= link_to 'Importar', former_students_upload_path, :class=>"btn-small"%></div>
-            <div class="col"><%= link_to 'Descargar', former_students_path(format: :csv), :class=>"btn-small"%></div>
-        </div>
+      <span class="table-title"><%= table_title %></span>
+      <div class="actions">
+        <div class="col"><a class="btn-small disabled table-button" table-button-tag="VER">Ver</a></div>
```

Comment:
Employers should not be able to see the former_students personal information

Reply:
Done.